README.md

```diff
@@ -34,15 +24,42 @@ export TF_VAR_AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
 
 ## Usage
 
-### Build
+### Prepare the environment
+
+1. Clone repo and boot the vagrant node with concourse:
+
+    ```
+git clone https://github.com/alphagov/paas-cf.git
+cd paas-cf
+vagrant up
+```
+
+2. Install your `fly` binary:
+
+    ```
+FLY_CMD_URL=http://192.168.100.4:8080/api/v1/cli?arch=amd64&platform=$(uname | tr '[:upper:]' '[:lower:]')
+sudo curl $FLY_CMD_URL -o /usr/local/bin/fly && \
+sudo chmod +x /usr/local/bin/fly
+```
+
+3. Login on the local concourse:
+
+     ```
+fly login --concourse-url http://192.168.100.4:8080 sync
+```
+
+### Deploy the deployer concourse
+
+`./concourse/scripts/create-deployer.sh`
```

Comment:
Could you change this to triple backticks? Change was lost in https://github.com/alphagov/paas-cf/pull/6#discussion_r47354327